source/transfer_list.rst

```diff
@@ -837,6 +837,47 @@ and reserved for use as a TPM Command Response Buffer interface.
      - Size of CRB.
 
 
+.. _dto_entry:
+
+DT overlay entry layout (XFERLIST_DT_OVERLAY)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Devicetree Overlay is defined in [DTO]_. The Devicetree Overlay TE contains
+the Device Overlay Note in the data section.
+The intent of the Devicetree Overlay entry is to modify the live devicetree in
+U-Boot or kernel.
```

Comment:
Can you please add a few more details about the lifefcycle here?
- Where does the overlay come from? 
- Where does the base devicetree come from?
- Why the overlay is needed in addition to the base devicetree (different types of hardware / variants?)
- Are the devicetrees signed? How does security work with this?
- When should the overlay be applied? In U-Boot, or Linux, or something else?
- Give an example of projects in the boot chain and how this would be used